packages/components-js/tests/e2e/puppeteer/specs/toast.e2e.ts

```diff
@@ -178,15 +178,15 @@ describe('toast-item', () => {
     const animationIn = await getElementStyle(toastItem, 'animation');
 
     expect(animationIn, 'for animationIn').toMatchInlineSnapshot(
-      '"0.6s cubic-bezier(0.45, 0, 0.55, 1) 0s 1 normal forwards running in"'
+      `"0s cubic-bezier(0, 0, 0.2, 1) 0s 1 normal forwards running in"`
```

Comment:
Is this correct from `0.6s` to now `0s`?

Reply:
It is now set to 0s because of the css variable `--p-animation-duration: 0s` in e2e tests..